tests/Hedgehog.Tests/ListGenTests.fs

```diff
@@ -0,0 +1,33 @@
+module Hedgehog.Tests.ListGenTests
+
+open Hedgehog
+open TestDsl
+
+
+let listGenTests = testList "ListGen tests" [
+    testCase "sequence is applicative" <| fun _ ->
+        let gen = Range.constant 0 1 |> Gen.int32
+        let listGen = [gen; gen]
+
+        let genList = listGen |> ListGen.sequence
+        let actual =
+            seq {
+                while true do
+                    let treeList = genList |> Gen.sampleTree 0 1 |> Seq.head
+                    if Tree.outcome treeList = [1; 1] then
+                        yield treeList
+            } |> Seq.head
```

Comment:
https://github.com/hedgehogqa/fsharp-hedgehog/pull/398#discussion_r776113580

;)

Reply:
Yep.  As we discussed, I am going to let this slide as well.